### PR TITLE
Randomize sound fxs

### PR DESCRIPTION
**Pull request by bytsbyts**

### CODE REVIEW

File: Assets/!/Prefabs/Player.prefab [hunk 1]
```diff
@@ -2005,22 +2005,71 @@ MonoBehaviour:
...
     clip: {fileID: 8300000, guid: 3d43d654cabcc80e386f62a4d5cf1775, type: 3}
+  smokeAudio:
+  - id: smoke1
+    clip: {fileID: 8300000, guid: d0af58fb14781365f846f7962bbda1ba, type: 3}
+  - id: smoke2
+    clip: {fileID: 8300000, guid: d06d8c27b8f6122e09813b7a7cd29cb8, type: 3}
+  - id: smoke3
+    clip: {fileID: 8300000, guid: e490fdb3bfa8e01e987d1c378055b7e0, type: 3}
+  - id: smoke4
+    clip: {fileID: 8300000, guid: 0f8bd2dccfaa1e856b341e5275186e1f, type: 3}
```
Comment (matias-gonz) [inline Assets/!/Prefabs/Player.prefab:2072]: The player doesnt use these

**Review by matias-gonz — CHANGES_REQUESTED** [review]

File: Assets/!/Scripts/Player/PlayerAudio.cs [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+    public Audio[] stepAudio;
+    public Audio[] dieScreamAudio;
+    public Audio[] bloodAudio;
+    public Audio[] crawlAudio;
+    public Audio[] bulletHitAudio;
+    public Audio[] jumpAudio;
+    public Audio[] fallDownAudio;
+    public Audio[] fireK98Audio;
+    public Audio[] k98CycleAudio;
+    public Audio[] smokeAudio;
```
Comment (matias-gonz) [inline Assets/!/Scripts/Player/PlayerAudio.cs:18]: rename to voice lines

**Review by matias-gonz — CHANGES_REQUESTED** [review]

File: Assets/!/Scripts/Player/PlayerAudio.cs (hunk 2 above)
Comment (bytsbyts) [reply Assets/!/Scripts/Player/PlayerAudio.cs:18]: Should we leave the name for now?

Comment (J3ns3its) [reply Assets/!/Scripts/Player/PlayerAudio.cs:18]: I think it is about renaming the Array. The Method will be renamed later.

Comment (bytsbyts) [reply Assets/!/Scripts/Player/PlayerAudio.cs]: renamed now

File: Assets/!/Scripts/Player/PlayerAudio.cs [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
+    public Audio[] stepAudio;
+    public Audio[] dieScreamAudio;
+    public Audio[] bloodAudio;
+    public Audio[] crawlAudio;
+    public Audio[] bulletHitAudio;
+    public Audio[] jumpAudio;
+    public Audio[] fallDownAudio;
+    public Audio[] fireK98Audio;
+    public Audio[] k98CycleAudio;
+    public Audio[] smokingSoldierVoiceLines;
```
Comment (matias-gonz) [inline Assets/!/Scripts/Player/PlayerAudio.cs:18]: Rename to `voiceLines`

**Review by matias-gonz — CHANGES_REQUESTED** [review]

**Review by matias-gonz — CHANGES_REQUESTED** [review]
Good!
One last thing, please change volume:
Run: 0.8
Walk: 0.5
Crouch: 0.35

**Comment by bytsbyts** [comment]
changed

**Review by matias-gonz — APPROVED** [review]

**Pull request merged**